R/reactives.R

```diff
@@ -400,7 +404,12 @@ Observable <- R6Class(
       on.exit(.running <<- wasRunning)
 
       ctx$run(function() {
-        result <- withVisible(try(shinyCallingHandlers(.func()), silent=TRUE))
+        result <- withCallingHandlers(withVisible(.func()),
+          error = function(cond) {
+            .visible <<- FALSE
+            .value <<- try(stop(cond), silent = TRUE)
```

Comment:
What's the purpose of the `try(stop(cond))`? Playing around with this, it looks like it'll convert `cond` from something like this:

```
> str(cond)
List of 2
 $ message: chr "foo"
 $ call   : language func()
 - attr(*, "class")= chr [1:3] "simpleError" "error" "condition"
```

into something like this:

```
> str(.value)
Class 'try-error'  atomic [1:1] Error in func() : foo

  ..- attr(*, "condition")=List of 2
  .. ..$ message: chr "foo"
  .. ..$ call   : language func()
  .. ..- attr(*, "class")= chr [1:3] "simpleError" "error" "condition"
```

If that's the goal, it would be nice to have a comment explaining it, because it's weird.